backend/apps/notifications/tasks.py

```
@@ -74,30 +91,38 @@ def send_bulk_email(payload):
         from apps.progress.services.pdf_report_service import PDFReportGenerator
         from django.contrib.auth import get_user_model
 
         User = get_user_model()
         user_email = recipients[0] if recipients else None
 
-        html_message = render_to_string("notifications/weekly_digest.html", data)
-        message = strip_tags(html_message)
+        try:
+            html_message = render_to_string("notifications/weekly_digest.html", data)
+            message = strip_tags(html_message)
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)